cli_tools/gce_vm_image_import/importer/bootable_disk_processor.go

```diff
@@ -97,6 +94,20 @@ func newBootableDiskProcessor(args ImportArguments) (processor, error) {
 	}, err
 }
 
+func findWorkflowFile(wfPath string, daisyWorkflowDir string) (string, error) {
+	for _, p := range []string{
+		wfPath,
+		path.Join(daisyWorkflowDir, wfPath),
+		path.Join(daisyWorkflowDir, "image_import", wfPath),
```

Comment:
Changed to use the original logic, which is:
1. Resolve custom workflow against the process's current working directory
2. Resolve non-custom workflow as `(daisyWorkflowDir, "image_import", wfPath)`

It was intended to add more flexibility, but you're right that it could have unintended consequences. Furthermore, in retrospect, it wasn't a great idea to make that change in this big review.